x-pack/plugin/otel-data/src/main/resources/component-templates/logs-otel@mappings.yaml

```diff
@@ -8,7 +8,7 @@ template:
     index:
       mode: logsdb
       sort:
-        field: [ "resource.attributes.host.name" ]
+        field: [ "@timestamp", "resource.attributes.host.name" ]
```

Comment:
Do we still need resource.attributes.host.name? Why was it there in the first place?

Reply:
From what i understand [the default for this setting is `@timestamp` and `host.name`](https://github.com/elastic/elasticsearch/blob/5750696069ef85be658c214a5d6281d083bd977c/server/src/main/java/org/elasticsearch/index/IndexSortConfig.java#L149-L151). I took this from @felixbarny in https://github.com/elastic/elasticsearch/issues/113397

So our goal is to just mimic the default.

I think alias does not help here, so we essentially just mimic the default, but in the otel case `host.name` is `resource.attributes.host.name`.

So I think we still want it there.

Reply:
The default sorts first by host name, then by timestamp. That’s an important detail that we should keep. The result is that the metadata for that host can be deduplicated because there are a a lot of the same values next to each other. 

If we sort by timestamp first, only documents with the exact same timestamp are sorted by host name.